tests/debug/compare_lib.py

```diff
@@ -132,12 +134,23 @@ def __init__(self, args):
 
 
 def deploy_to_pipeline(args) -> None:
+    with open(
+        os.path.join(args.artifact_path, "params", "mlc-chat-config.json"), "r"
+    ) as f:
+        config = json.load(f)
+
     primary_device = tvm.device(args.primary_device)
     const_params = utils.load_params(args.artifact_path, primary_device)
     state = TestState(args)
-    tokenizer = AutoTokenizer.from_pretrained(
-        os.path.join(args.artifact_path, "params"), trust_remote_code=True
-    )
+
+    if config["model_category"] == "llama":
+        tokenizer = LlamaTokenizer.from_pretrained(
+            os.path.join(args.artifact_path, "params"), trust_remote_code=True
+        )
+    else:
+        tokenizer = AutoTokenizer.from_pretrained(
+            os.path.join(args.artifact_path, "params"), trust_remote_code=True
+        )
```

Comment:
When the model is LLaMA-structured, using `LlamaTokenizer` is faster than `AutoToeknizer`.